### ISSUE TITLE
Wallet is not updating

### ISSUE THREAD

**Issue by Matt98camaro** [oscarguindzberg/coproc#16]
I went to log into my wallet and it came up with "could not update wallet: Prosight error"
So then I uninstalled app and reinstalled and tried to put in my 12 word phrase and it would not import or anyting I cannot see my funds any longer please help

**Comment by Costalkarma** [comment]
I'm having the same issue

**Comment by Matt98camaro** [comment]
I was hoping someone could help us from tech support.  😡

**Comment by Matt98camaro** [comment]
Can anyone help with this issue?? Getting really frustrated???

**Comment by MPilarchp** [comment]
Veo que el soporte no responde. A mi me ha pasado lo mismo que a vosotros. Puse incidencia hace dos días. Sin respuesta aún...

**Comment by Matt98camaro** [comment]
Sí, muy frustrante. Estoy enfadado por todo el asunto. definitivamente nos estafaron...Sent from my Sprint Samsung Galaxy S9.
-------- Original message --------From: MPilarchp <notifications@github.com> Date: 8/13/20  10:00 AM  (GMT-05:00) To: procommerce-io/coproc <coproc@noreply.github.com> Cc: Matt98camaro <mpetersen768@gmail.com>, Author <author@noreply.github.com> Subject: Re: [procommerce-io/coproc] Wallet is not updating (#16) 

Veo que el soporte no responde. A mi me ha pasado lo mismo que a vosotros. Puse incidencia hace dos días. Sin respuesta aún...

**Comment by Roobz77** [comment]
I think Proc is dead, all I get is a network error  as well... trying to retrieve my coins, can some one help with this issue?? Talk about a all time shitcoin.

**Comment by Matt98camaro** [comment]
I think they're Done brother, I have lost mine as well and no one is giving me any help to retrieve them pretty much just lost my money. I'm giving up and moving on now...🤷‍♂️Sent from my Sprint Samsung Galaxy S9.
-------- Original message --------From: Roobz77 <notifications@github.com> Date: 9/10/20  4:37 PM  (GMT-05:00) To: procommerce-io/coproc <coproc@noreply.github.com> Cc: Matt98camaro <mpetersen768@gmail.com>, Author <author@noreply.github.com> Subject: Re: [procommerce-io/coproc] Wallet is not updating (#16) 
I think Proc is dead, all I get is a network error  as well... trying to retrieve my coins, can some one help with this issue?? Talk about a all time shitcoin.
